localstack/services/es/es_api.py

```diff
@@ -165,6 +133,16 @@ def error_response(error_type, code=400, message="Unknown error."):
     return response, code
 
 
+def get_domain_arn(domain_name: str, region: str = None, account_id: str = None) -> str:
```

Comment:
nit: could also reuse `aws_stack.elasticsearch_domain_arn(..)` here. (note to self: ideally we should extract all the ARN util functions out of `aws_stack.py` into a separate file..).

Reply:
good point